src/OpenRoad.cc

```diff
@@ -469,12 +462,10 @@ void OpenRoad::readDb(const char* filename)
 
 void OpenRoad::writeDb(const char* filename)
 {
-  std::ofstream stream;
-  stream.exceptions(std::ifstream::failbit | std::ifstream::badbit
-                    | std::ios::eofbit);
-  stream.open(filename, std::ios::binary);
+  utl::StreamHandler stream_handler(
+      filename, std::ofstream::badbit | std::ios::eofbit, std::ios::binary);
```

Comment:
remove "badbit" from arguments and eofbit, just add them inside the StreamHandler implementation as they are required for correct operation of StreamHandler.